.travis.yml

```diff
@@ -16,13 +19,17 @@ install:
   - git clone https://github.com/nf-core/tools.git /tmp/nf-core-tools
   - cd /tmp/nf-core-tools
   - pip install --user -e .
-  # Reset
+  # Make test directories
+  - mkdir ${TRAVIS_BUILD_DIR}/tests
   - cd ${TRAVIS_BUILD_DIR}/tests
 
 env:
-  - NXF_VER=0.27.6
-  - ''
+  - NXF_VER=0.30.1
```

Comment:
Do we really need v0.30.1? The pipeline should work with v0.30.0 right?

Reply:
Yup :-)